.ci/travis/travis.sh

```diff
@@ -387,6 +387,21 @@ check-missing-pitests)
   exit $fail
   ;;
 
+verify-no-exception-configs)
+  wget -q \
+    https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/checks-nonjavadoc-error.xml
+  wget -q \
+    https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/checks-only-javadoc-error.xml
+  xmlstarlet sel --net --template -m .//module -v "@name" \
+    -n checks-nonjavadoc-error.xml -n checks-only-javadoc-error.xml \
+    | grep -vE "Filter" | grep -v "^$" \
+    | sort | uniq | sed "s/Check$//" > web.txt
+  xmlstarlet sel --net --template -m .//module -v "@name" -n config/checkstyle_checks.xml \
+    | grep -vE "Filter|ImportControl" | grep -v "^$" \
```

Comment:
ImportControl and Filters are ignored because it requires an external file.

Reply:
please do 
```
...
MODLUES_WITH_EXTERNAL_FILES="Filter|ImportControl" \
| grep -vE $MODLUES_WITH_EXTERNAL_FILES | grep -v "^$" \
```

to self explain this tiny nuance.

Reply:
Done.